web/angular-wallet/src/app/setup/account/create-active/pin/pin.component.ts

```diff
@@ -17,20 +17,14 @@ export class AccountCreatePinComponent implements OnInit {
         private createService: CreateService,
         private notificationService: NotifierService,
     ) { }
-
-    public ngOnInit() {
-
-    }
-
-    public ngOnDestroy() {
-
-    }
-
-    public back() {
+    public back(): void {
         this.createService.setStepIndex(1);
     }
 
-    public finish(pin: string) {
+    public finish(pin: string): void {
+        if (!pin || !pin.length) {
+            return;
+        }
```

Comment:
Don't let users through without a PIN, otherwise they will be unable to send burst and doomed like me!

Reply:
:eyes: